### PR TITLE
Add menu swaps for Brimhaven and Rimmington

### PR DESCRIPTION
**Pull request by BrianJMRogers**
Screenshot 25
19be6526b57210ffbded0603112d4547
b58287183e1107d9c8bbecf2a3b582f1
Screenshot 24

### CODE REVIEW

**Comment by BrianJMRogers** [comment]
@Hydrox6 I've made the requested changes =]

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 1]
```diff
@@ -92,6 +92,16 @@ default boolean swapBones()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "swapCaptainBarnaby",
```
Comment (Hydrox6) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: There's an extra tab on these 3 lines

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/CaptainBarnaby.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package net.runelite.client.plugins.menuentryswapper;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CaptainBarnaby
```
Comment (Hydrox6) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/CaptainBarnaby.java]: This should probably be named something like `CaptainBarnabyMode`, to be more descriptive

**Comment by BrianJMRogers** [comment]
@Hydrox6 Made those three changes

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/CaptainBarnabyMode.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package net.runelite.client.plugins.menuentryswapper;
```
Comment (Hydrox6) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/CaptainBarnabyMode.java]: Needs a license. Have a look at other files for the format to use

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 4]
```diff
@@ -92,6 +92,16 @@ default boolean swapBones()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "swapCaptainBarnaby",
+		name = "Captain Barnaby",
+		description = "Swap Talk-to with Brimhaven or Rimmington"
```
Comment (Hydrox6) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: Could either mention Ardougne, or could just be more general to stay shorter

**Comment by BrianJMRogers** [comment]
@Hydrox6 boom

**Pull request closed without merging**